frontend/src/pages/Mesh/target/TargetPanelNode.tsx

```diff
@@ -23,15 +26,31 @@ export const TargetPanelNode: React.FC<TargetPanelNodeProps<MeshNodeData>> = (
   }
 
   const data = node.elem.getData()!;
-
   return (
     <div id="target-panel-node" className={classes(panelStyle, targetPanelStyle)}>
       <div className={panelHeadingStyle}>{renderNodeHeader(data, { nameOnly: isExternal(data.cluster) })}</div>
       <div className={panelBodyStyle}>
         <span>{t('Version: {{version}}', { version: data.version || t(UNKNOWN) })}</span>
 
         {targetPanelHR}
-
+        {data.infraName === TEMPO && props.tracingStats?.hitRate && (
+          <>
+            <div style={{ display: 'table' }}>
+              Cache:
+              <div style={{ marginLeft: '20px' }}>
+                <span style={{ color: 'rgb(25, 118, 116)', fontWeight: 'bold' }}>Hit rate:</span>{' '}
+                {props.tracingStats.hitRate}
+              </div>
+              {props.tracingStats.size && (
+                <div style={{ marginLeft: '20px' }}>
+                  <span style={{ color: 'rgb(25, 118, 116)', fontWeight: 'bold' }}>Size:</span>{' '}
+                  {props.tracingStats.size}
+                </div>
+              )}
+            </div>
+            {targetPanelHR}
+          </>
+        )}
```

Comment:
I didn't know Kiali also emits its own prom metrics. So, probably can be done in the same way and store this value as a metric. 
I though the Mesh page because that is intended for admins, which can be also a good place to made them know that he cache is active and if it is useful or not.